sdk/plugin/pb/translation.go

```diff
@@ -491,13 +492,14 @@ func LogicalResponseToProtoResponse(r *logical.Response) (*Response, error) {
 	}
 
 	return &Response{
-		Secret:   secret,
-		Auth:     auth,
-		Data:     string(buf[:]),
-		Redirect: r.Redirect,
-		Warnings: r.Warnings,
-		WrapInfo: wrapInfo,
-		Headers:  headers,
+		Secret:    secret,
+		Auth:      auth,
+		Data:      string(buf[:]),
+		Redirect:  r.Redirect,
+		Warnings:  r.Warnings,
+		WrapInfo:  wrapInfo,
+		Headers:   headers,
+		MountType: r.MountType,
```

Comment:
Do we need a test that does forwarding, to verify that this translation is done?

Reply:
Good call! I missed updating the test for this

Reply:
Updating that test will validate that ENT-style request forwarding works, where we serialize responses via protobuf.  We still need something to exercise HTTPResponseToLogicalResponse, for CE-style request forwarding.  I think your tests in kv_test.go would be a good place to do this, using a multi-node cluster with perf standbys disabled and doing requests via the standby.